core-java-modules/core-java-21/src/test/java/com/baeldung/java21/EmojiSupportUnitTest.java

```diff
@@ -0,0 +1,69 @@
+package com.baeldung.java21;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
```

Comment:
Use the JUnit5 imports not the JUnit4 ones

Reply:
review incorporated.